### ISSUE TITLE
Opening Keyboard in child view pushes parent view up - Android Fullscreen mode

### ISSUE THREAD

**Issue by antoine92190** [ionic-team/ionic#13140]
**Ionic version:**  (check one with "x")
[ ] **2.x**
[x] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

**Current behavior:**
- My app is opened on an Android device in fullscreen mode (available on Samsung S8 for example)
- I have a Home Page with a list of items
- I have a Child Page with an `ion-input`

1. When opening the Child Page from the Home Page using `navCtrl.push()`, then clicking on the input, the keyboard and the Android action bar appear.

2. When I have finished writing, the keyboard and the Android action bar disappear.

=> This action makes it so the Home Page view is pushed up, and I cannot not scroll at the top of this page anymore.

**Expected behavior:**
The usage of the keyboard in Android fullscreen mode does not pushes the previous view up.

**Steps to reproduce:**
- Use the repo below and get an Android device where you can put the app in fullscreen mode
- Build and open the project on your Android device
- When the app opens, click on "Item 1" to open the Keyboard Page
- Click on the input to make the keyboard appear, then close it
- Go back to the Home Page using the back button
- You will notice that the view is pushed up and you cannot access "Item 1" anymore

**Related code:**

https://github.com/antoine92190/keyboard-fullscreen

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.13.0
    ionic (Ionic CLI) : 3.13.0

local packages:

    @ionic/app-scripts : 3.0.0
    Ionic Framework    : ionic-angular 3.7.1

System:

    Node : v6.11.4
    npm  : 3.10.10 
    OS   : macOS High Sierra

Misc:

    backend : pro
```

**Comment by LuizTokuhara** [comment]
I have a similar problem, if you go to a child page (navCtrl.push()) that has a prompt alert, when go back to home view the scrolling will get stuck and you cannot slide the screen up or down.

Tested with android device (6.0.1).

cli packages: (/node_modules)

    @ionic/cli-utils  : 1.12.0
    ionic (Ionic CLI) : 3.12.0

global packages:

    cordova (Cordova CLI) : 7.0.1 

local packages:

    @ionic/app-scripts : 3.0.0
    Cordova Platforms  : android 6.2.3
    Ionic Framework    : ionic-angular 3.7.0

System:

    Android SDK Tools : 26.0.2
    ios-sim           : 6.0.0 
    Node              : v8.1.3
    npm               : 5.3.0 
    OS                : macOS Sierra
    Xcode             : Xcode 8.3.3 Build version 8E3004b 

Misc:

    backend : pro

**Comment by Gamerakki** [comment]
hi,

can u try the native-keyboard plugin from the ionic framework . here is the link: https://ionicframework.com/docs/native/native-keyboard/ this might solve the problem. The keyboard issue there in ionic for a long time. After doing research i found that this plugin solve most of such problem .

**Comment by antoine92190** [comment]
Related to #13028

**Comment by polfernandez** [comment]
Having the same issue. With ionic-angular 3.7.1, the entire screen moves up when ionic-native keyboard shows (from a ion-textarea input).

on all iOS devices. for Android works fine

**Comment by antoine92190** [comment]
Fixed in ionic 3.8.0
